1209/F3FC/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: dRonin support for Lumenier Lux flight controller
```

Comment:
Thanks for running this project and the feedback!  I've pushed a new attempt.

Just for background, dRonin supports a lot of flight controllers which have pre-existing USB vendor /product IDs.  This is a port to a third-party controller which doesn't have a valid pair, and when running our firmware we need it to be recognizable by our ground software.

We're really thankful if your project will let us obtain a valid PID for this scenario.